packages/mui-material/src/Unstable_TrapFocus/FocusTrap.tsx

```diff
@@ -164,23 +183,25 @@ function FocusTrap(props: FocusTrapProps): React.JSX.Element {
     }
 
     const doc = ownerDocument(rootRef.current);
+    const openFocusElement = getTabbable(rootRef.current, true, true)?.[0] ?? rootRef.current;
```

Comment:
This is internal component so maybe we don't need to document in the types that `getTabbable` prop now accepts three parameters: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Unstable_TrapFocus/FocusTrap.types.ts#L13, but it would be better to add it.

Reply:
added